lib/modules/versioning/swift/index.spec.ts

```diff
@@ -108,8 +108,8 @@ describe('modules/versioning/swift/index', () => {
 
   it.each`
     currentValue           | rangeStrategy | currentVersion | newVersion  | expected
-    ${'1.2.3'}             | ${'auto'}     | ${'1.2.3'}     | ${'1.2.4'}  | ${'1.2.3'}
-    ${'v1.2.3'}            | ${'auto'}     | ${'v1.2.3'}    | ${'v1.2.4'} | ${'v1.2.3'}
```

Comment:
Can you please point to the docs, where it's mentioned that `v` in version is not legal or something similar.

Reply:
[Swift package documentation for version values](https://docs.swift.org/package-manager/PackageDescription/PackageDescription.html#version)